src/components/Tooltip/BaseTooltip/index.tsx

```diff
@@ -87,8 +87,8 @@ function Tooltip({children, shouldHandleScroll = false, ...props}: TooltipProps,
         [children, updateTargetAndMousePosition],
     );
 
-    // Skip the tooltip and return the children if the device does not support hovering
-    if (!hasHoverSupport) {
+    // If the device doesn't support hover or the element isn't focused, skip rendering the tooltip and just return the children.
```

Comment:
The focused hook for navigation is not for an element.
```suggestion
    // Skip the tooltip and return the children if the device does not support hovering or if navigation does not focus.
```

Reply:
it's done